eox_core/tests/tutor/integration_test_users.py

```diff
@@ -0,0 +1,173 @@
+"""
+Integration test suite for the Users API.
+
+This suite performs multiple http requests to guarantee
+that the Users API is behaving as expected on a live server.
+"""
+
+from django.contrib.auth.models import User
+from django.contrib.sites.models import Site
+from django.http import HttpResponse
+from django.test import TestCase, override_settings
+from django.urls import reverse
+
+try:
+    from eox_tenant.models import Route, TenantConfig
+except ImportError:
+    Route = None
+    TenantConfig = None
+from oauth2_provider.models import Application
+from rest_framework import status
+
+CLIENT_ID = "apiclient"
+CLIENT_SECRET = "apisecret"
+
+
+def create_oauth2_client(user: User) -> None:
+    """
+    Create a new OAuth2 client.
+
+    Args:
+        user (User): The user that will own the client.
+    """
+    Application.objects.create(
+        name="eox-core-app",
+        client_id=CLIENT_ID,
+        client_secret=CLIENT_SECRET,
+        client_type=Application.CLIENT_CONFIDENTIAL,
+        authorization_grant_type=Application.GRANT_CLIENT_CREDENTIALS,
+        user=user,
+    )
+
+
+def create_admin_user() -> User:
+    """
+    Create a new admin user.
+
+    Returns:
+        User: The admin user.
+    """
+    return User.objects.create_superuser("eox-core-admin", "eox-core@mail.com", "p@$$w0rd")
+
+
+@override_settings(ALLOWED_HOSTS=["*"], SITE_ID=2)
+class TestUsersAPIIntegration(TestCase):
+    """Integration test suite for the Users API"""
+
+    def setUp(self):
+        """Set up the test suite"""
+        self.path = reverse("eox-core:eox-api:eox-api:edxapp-user")
+        self.admin_user = create_admin_user()
+        create_oauth2_client(self.admin_user)
+        self.tenant_x = self.create_tenant("tenant-x")
+        self.tenant_y = self.create_tenant("tenant-y")
+
+    @override_settings(EOX_CORE_USERS_BACKEND="eox_core.edxapp_wrapper.backends.users_m_v1")
```

Comment:
can the path be saved in a variable like CLIENT_ID and replaced here and after?

Reply:
You're right!, done: https://github.com/eduNEXT/eox-core/commit/5ca9726d6a2b9214bf50a158d8054282fa2a2e6b